draft-ietf-masque-connect-udp.md

```diff
@@ -276,32 +276,53 @@ the client (e.g., "connect-udp-version: 2"). Sending this header is RECOMMENDED
 but not required.
 
 
+# Context Identifiers {#context-id}
+
+This protocol allows future extensions to exchange HTTP Datagrams which carry
+different semantics from UDP payloads. Some of these extensions can augment UDP
+payloads with additional data, while others can exchange data that is completely
+separate from UDP payloads. In order to accomplish this, all HTTP Datagrams
+associated with UDP Proxying request streams start with a context ID, see
+{{format}}.
+
+Context IDs are 62-bit integers (0 to 2<sup>62</sup>-1). The context ID value of
+zero is reserved for UDP payloads, while non-zero values are dynamically
+allocated: non-zero even-numbered context IDs are client-initiated, and
+odd-numbered context IDs are server-initiated. Extensions can use the next
+available context ID for their own purposes. Note that, once allocated, any
+context ID can be used by both client and server - only allocation carries
+separate namespaces to avoid requiring synchronization. Additionally, note that
+the context ID namespace is tied to a given HTTP request: it is possible for the
+same numeral context ID to be used simultaneously in distinct requests.
+
+
 # HTTP Datagram Payload Format {#format}
 
 When associated with UDP proxying request streams, the HTTP Datagram Payload
 field of HTTP Datagrams (see {{HTTP-DGRAM}}) carries the following semantics:
```

Comment:
Pre-existing text, but this is *format*, not semantics.  The semantics follow (i.e., meetings will continue until morale improves).

Reply:
Agreed. Fixed in the 2021_design_team_consensus branch